ipaserver/install/dsinstance.py

```diff
@@ -24,14 +24,20 @@
 import shutil
 import pwd
 import os
-import re
 import time
 import tempfile
 import stat
 import fnmatch
 
 import ldap
 
+from lib389 import DirSrv
```

Comment:
Please sort this block alphanumerical and remove the surplus newline between the lib389 block and the ipa import block.

Reply:
Fixed.